authorizers/iconik/authorizer.ts

```diff
@@ -0,0 +1,88 @@
+import { IconikService } from '@workflowwin/iconik-api';
+import { CustomActionPayload } from '@workflowwin/iconik-api/dist/src/assets/assets-methods';
+import { AppError, CommonErrors } from '@helper/app-error';
+import { getEnv } from '@helper/environment';
+import { log } from '@helper/logger';
+import { IconikContext } from './context';
+
+type Payload = CustomActionPayload & { auth_token?: string };
+
+/**
+ * iconik doesn't send Auth-Token in headers but sends it in the body
+ * So we need to check it in the Lambda function instead using API Gateway authorizers
+ * How to use this authorizer:
```

Comment:
@azhivaeva-flo  It means we should use it in a feature function. Why not as a custom auth in yml? I saw your message in readme but can't see example as a custom auth in yml.